### PR TITLE
android_jni: Include libyuv in version string only when available

### PR DESCRIPTION
**Pull request by vigneshvg**

### CODE REVIEW

**Review by y-guyon — APPROVED** [review]

File: android_jni/avifandroidjni/src/main/jni/libavif_jni.cc [hunk 1]
```diff
@@ -376,10 +376,17 @@ FUNC(jstring, resultToString, jint result) {
 FUNC(jstring, versionString) {
   char codec_versions[256];
   avifCodecVersions(codec_versions);
+  char libyuv_version[64];
+  if (avifLibYUVVersion() > 0) {
+    snprintf(libyuv_version, sizeof(libyuv_version), " libyuv: %d.",
+             avifLibYUVVersion());
+  } else {
+    *libyuv_version = 0;
```
Comment (wantehchang) [inline android_jni/avifandroidjni/src/main/jni/libavif_jni.cc:384]: Nit: The following looks nicer:

```
    libyuv_version[0] = '\0';
```

File: android_jni/avifandroidjni/src/main/jni/libavif_jni.cc [hunk 2]
```diff
@@ -376,10 +376,17 @@ FUNC(jstring, resultToString, jint result) {
 FUNC(jstring, versionString) {
   char codec_versions[256];
   avifCodecVersions(codec_versions);
+  char libyuv_version[64];
+  if (avifLibYUVVersion() > 0) {
+    snprintf(libyuv_version, sizeof(libyuv_version), " libyuv: %d.",
```
Comment (wantehchang) [inline android_jni/avifandroidjni/src/main/jni/libavif_jni.cc:381]: Nit: Use `%u` instead of `%d` because `avifLibYUVVersion()` returns `unsigned int`.

**Review by wantehchang — COMMENTED** [review]
LGTM.

File: android_jni/avifandroidjni/src/main/jni/libavif_jni.cc (hunk 1 above)
Comment (vigneshvg) [reply android_jni/avifandroidjni/src/main/jni/libavif_jni.cc:384]: https://github.com/AOMediaCodec/libavif/pull/1614

File: android_jni/avifandroidjni/src/main/jni/libavif_jni.cc (hunk 2 above)
Comment (vigneshvg) [reply android_jni/avifandroidjni/src/main/jni/libavif_jni.cc:381]: https://github.com/AOMediaCodec/libavif/pull/1614

**Pull request merged**